backend/btrixcloud/models.py

```diff
@@ -1180,7 +1180,6 @@ class S3Storage(BaseModel):
     secret_key: str
     access_endpoint_url: str
```

Comment:
While I'm thinking about documentation, I don't think we currently have `access_endpoint_url` documented at all in the Self-Hosting docs, seems like a good addition.